store/postgres/tests/relational.rs

```diff
@@ -1000,6 +1000,10 @@ fn check_find() {
                 vec!["3"],
                 user_query().filter(EntityFilter::Equal("userSearch".into(), "Shaq:*".into())),
             )
+            .check(
+                vec!["1"],
+                user_query().filter(EntityFilter::Equal("userSearch".into(), "Jono a".into())),
+            )
```

Comment:
Yes, I think this is exactly the right kind of test (I assume it was failing before the changes above)

We should try and see if there are other funky search terms that cause `to_tsquery` to fail. It's probably also worth checking with @fordN who wrote this initially because IIRC it is possible to put some of the text search operators into the search string.

Reply:
cool, will do. There are definitely a few more use cases as well

Reply:
And yep it was failing before the code changes :)